### PR TITLE
Release/1.9.0

### PR DESCRIPTION
**Pull request by gedge**
### What

- kafka TLS (MSK) ready - @justinpjose 
- log v2 - @stmuk 
- elasticsearch signing - @justinpjose 

### How to review

ensure the above is correct

### Who can review

you?

### CODE REVIEW

File: .nancy-ignore [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+# Dependencies used by github.com/gopherjs/gopherjs@v0.0.0-20210803090616-8f023c250c89
```
Comment (nshumoogum) [inline .nancy-ignore:3]: Do we have bug stories in the backlog to fix these in the future? If not then let me know and we can create them

File: ci/build.yml [hunk 2]
```diff
@@ -6,7 +6,7 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: 1.16.3
+    tag: 1.16.7
```
Comment (nshumoogum) [inline ci/build.yml]: As this is a release, we don't need to do anything here now - but general fyi we should be moving to go version 1.17.1 now :)

File: main.go [hunk 3]
```diff
@@ -49,55 +49,66 @@ func run(ctx context.Context) error {
...
 
 	hierarchyClient := hierarchy.New(cfg.HierarchyAPIURL)
-	elasticSearchClient := elasticsearch.NewClient(cfg.ElasticSearchAPIURL, cfg.SignElasticsearchRequests, cfg.MaxRetries)
+
+	var awsSDKSigner *esauth.Signer
+	if cfg.SignElasticsearchRequests {
+		awsSDKSigner, err = esauth.NewAwsSigner("", "", cfg.AwsRegion, cfg.AwsService)
+		if err != nil {
+			log.Error(ctx, "failed to create aws v4 signer", err)
+			os.Exit(1)
```
Comment (nshumoogum) [inline main.go]: This should return err
```suggestion
			return err
```

**Review by nshumoogum — CHANGES_REQUESTED** [review]

File: .nancy-ignore (hunk 1 above)
Comment (gedge) [reply .nancy-ignore:3]: not to my knowledge - happy to be shown how

File: ci/build.yml (hunk 2 above)
Comment (gedge) [reply ci/build.yml]: fully aware of that, but the work in `develop` was done before that came out, AIUI

File: main.go (hunk 3 above)
Comment (gedge) [reply main.go]: https://github.com/ONSdigital/dp-dimension-search-builder/pull/52

Comment (gedge) [reply main.go]: this is now addressed

**Review by nshumoogum — APPROVED** [review]

**Pull request merged**